scipy/_lib/_util.py

```diff
@@ -446,10 +447,13 @@ def wrapper(*args, **kwargs):
             if 'rng' in parameter_names:
                 _type = "{None, int, `numpy.random.Generator`}, optional"
                 _desc = _rng_desc.replace("{old_name}", old_name)
-                _rng_parameter_doc = Parameter('rng', _type, [_desc])
+                old_doc = doc['Parameters'][parameter_names.index('rng')].desc
+                old_doc_keep = old_doc[old_doc.index("") + 1:] if "" in old_doc else []
+                new_doc = [_desc] + old_doc_keep
+                _rng_parameter_doc = Parameter('rng', _type, new_doc)
                 doc['Parameters'][parameter_names.index('rng')] = _rng_parameter_doc
-            doc = str(doc).split("\n", 1)[1]  # remove signature
-            wrapper.__doc__ = str(doc)
+                doc = str(doc).split("\n", 1)[1]  # remove signature
+                wrapper.__doc__ = str(doc)
```

Comment:
Not an essential change; it's just something I noticed was not needed unless the `if` condition is satisfied.